### PR TITLE
[cli] migrate mysqlctl and mysqlctld to cobra

### PR DESCRIPTION
**Pull request by ajm188**
## Description

Less straightforward, `mysqlctl`'s subcommands lend a structure better to `vtctldclient` than the others we've moved so far.

## Related Issue(s)

#13918 

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required: https://github.com/vitessio/website/pull/1582

### CODE REVIEW

File: go/flags/endtoend/mysqlctl.txt [hunk 1]
```diff
@@ -62,9 +66,9 @@ Global flags:
       --log_rotate_max_size uint                                    size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
       --logtostderr                                                 log to standard error instead of files
       --max-stack-size int                                          configure the maximum stack size in bytes (default 67108864)
-      --mysql_port int                                              MySQL port (default 3306)
+      --mysql_port int                                              MySQL port. (default 3306)
```
Comment (deepthi) [inline go/flags/endtoend/mysqlctl.txt:69]: Why do some flags have a period at the end of the help text and others don't?

**Review by deepthi — APPROVED** [review]

File: go/flags/endtoend/mysqlctl.txt (hunk 1 above)
Comment (ajm188) [reply go/flags/endtoend/mysqlctl.txt:69]: mistake (meant to update all of the local ones)

Comment (ajm188) [reply go/flags/endtoend/mysqlctl.txt:69]: oh i see what you're asking. these flags get defined local to `mysqlctl`, so i made them all (mostly, missed a few which i'm fixing now) proper sentences. the ones defined all over the codebase are bit less consistent

**Review by frouioui — APPROVED** [review]

**Comment by deepthi** [comment]
Upgrade/Downgrade test failures may be legit. They are saying "error in mysqlctl shutdown", and they don't fail on #14007

**Comment by ajm188** [comment]
yeah, added [514c946](https://github.com/vitessio/vitess/pull/13946/commits/514c946457c8c54dbbb1f6d5890c337932811c48) which i forgot to push lol

**Comment by ajm188** [comment]
all better now @deepthi

**Pull request merged**